tutorials/deploy-local/1.deploy-local.ipynb

```diff
@@ -0,0 +1,726 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7861af8f",
+   "metadata": {},
+   "source": [
+    "Then ensure you can send a post request to the service:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "c0dac0e2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "test is {'query': 'What color is the fox', 'context': 'The quick brown fox jumped over the lazy dog.'}\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "uri = service.scoring_uri\n",
+    "\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "data = {\"query\": \"What color is the fox\", \"context\": \"The quick brown fox jumped over the lazy dog.\"}\n",
+    "data = json.dumps(data)\n",
+    "response = requests.post(uri, data=data, headers=headers)\n",
+    "print(response.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc9aab6b",
+   "metadata": {},
+   "source": [
+    "## Re-deploy to cloud\n",
+    "\n",
+    "Once you've confirmed your service works locally and chosen a remote compute target, you are ready to deploy to the cloud.\n",
+    "\n",
+    "Change your deploy configuration to correspond to the compute target you've chosen, in this case Azure Container Instances."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "e0a4ce66",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.webservice import AciWebservice\n",
+    "\n",
+    "deployment_config = AciWebservice.deploy_configuration(cpu_cores = 0.5, memory_gb = 1, auth_enabled=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4828de68",
+   "metadata": {},
+   "source": [
+    "Deploy your service again"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "51029f65",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Tips: You can try get_logs(): https://aka.ms/debugimage#dockerlog or local deployment: https://aka.ms/debugimage#debug-locally to debug if deployment takes longer than 10 minutes.\n",
+      "Running\n",
+      "2021-04-30 18:30:38+05:30 Creating Container Registry if not exists.\n",
+      "2021-04-30 18:30:39+05:30 Registering the environment.\n",
+      "2021-04-30 18:30:40+05:30 Use the existing image.\n",
+      "2021-04-30 18:30:40+05:30 Generating deployment configuration.\n",
+      "2021-04-30 18:30:42+05:30 Submitting deployment to compute..\n",
+      "2021-04-30 18:31:30+05:30 Checking the status of deployment testservice1..\n",
+      "2021-04-30 18:32:53+05:30 Checking the status of inference endpoint testservice1.\n",
+      "Succeeded\n",
+      "ACI service creation operation finished, operation \"Succeeded\"\n"
+     ]
+    }
+   ],
+   "source": [
+    "service = Model.deploy(ws, \"myservice\", [model], inference_config, deployment_config)\n",
```

Comment:
Can you use inf_config instead of inference_config so that we deploy the real model to ACI?

Reply:
Done

Reply:
I'm still seeing inf_config here?